grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/ArmeriaMessageDeframer.java

```diff
@@ -83,32 +83,37 @@
 
     private static final int HEADER_LENGTH = 5;
     private static final int COMPRESSED_FLAG_MASK = 1;
-    private static final int RESERVED_MASK = 0xFE;
+    private static final int RESERVED_MASK = 0x7E;
+    // Valid type always has 0s for middle bits.
+    private static final int UNINITIALIED_TYPE = 0xFF;
```

Comment:
Maybe use `-1` which is even more impossible?

Reply:
Isn't -1 FF? Is it clearer using the decimal?

Reply:
You are using `ByteBuf.readUnsignedByte()` which never returns a negative value, so `0xFF` and `-1` are different. :wink:

Reply:
Ah thanks that is better then!